README.md

```diff
@@ -190,7 +190,19 @@ g4ndl@4.5          hwloc@2.0.2              libxext@1.3.3         pythia8@8240
 
 ### 5. Use
 
-TODO
+Symbolic links to packages in install tree can be placed to a common installation prefix using view command.
+This will create a directory, which can be used as SIMPATH in order to build experiment-specific frameworks.
+View is created by following command:
+
+Linux
+```
+spack -C ./config view --verbose --dependencies true symlink -i (YOUR_SIMPATH) fairroot
```

Comment:
```suggestion
[jun19] $ spack -C ./config view --verbose --dependencies true symlink -i (YOUR_SIMPATH) fairroot
```

To show the view must be created while the previously created environment is activated, otherwise I wouldn't understand how this works?